.github/workflows/update-docs.yml

```diff
@@ -20,6 +20,9 @@ on:
 
 permissions: read-all
 
+permissions:
```

Comment:
`permissions` key is already set at line 21 and gets repeated at line 23. This will likely cause a syntax error in the YAML file. 

I stumbled upon this PR since it mentioned https://app.stepsecurity.io/. I am the maintainer of that [tool](https://github.com/step-security/secure-workflows). I did check and it should not add `permissions` if they are already defined. 

@dthaler can you please confirm this `permissions` key was added by the tool? If so, I will dig deeper to find the issue to prevent duplication of the `permissions` key. Thanks!

Reply:
Hmm, this was the first file I did and I must have done it differently since when I repeat the exercise now I don't get the duplication, so I think the tool is ok.